docs/builds.rst

```diff
@@ -133,57 +133,7 @@ but you can change that using a :doc:`config-file/index`.
    If you want to know the specific version of a package that is installed in the image
    you can check the `Ubuntu package search page <https://packages.ubuntu.com/>`__.
 
-2.0 (stable)
-~~~~~~~~~~~~
-
-:O.S: Ubuntu 16.04
-:Conda: Miniconda 4.3.31
-:Python:
-  * ``m2crypto``
-  * ``matplolib``
-  * ``numpy``
-  * ``pandas``
-  * ``pip``
-  * ``scipy``
-:Other packages:
-  * ``doxygen``
-  * ``graphviz``
-  * ``libevent``
-  * ``libjpeg``
-  * ``libxml2-dev``
-  * ``libxslt1.1``
-  * ``pandoc``
-  * ``textlive-full``
-
-`More details <https://github.com/rtfd/readthedocs-docker-images/blob/releases/2.x/Dockerfile>`__
-
-3.0 (latest)
-~~~~~~~~~~~~
-
-:O.S: Ubuntu 16.04
-:Conda: Miniconda 4.4.10
-:Python:
-  * ``matplolib``
-  * ``numpy``,
-  * ``pandas``
-  * ``pip``
-  * ``scipy``
-:JavaScript:
-  * ``jsdoc``
-  * ``nodejs``
-  * ``npm``
-:Other packages:
-  * ``doxygen``
-  * ``libevent-dev``
-  * ``libgraphviz-dev``
-  * ``libjpeg``
-  * ``libxml2-dev``
-  * ``libxslt1-dev``
-  * ``pandoc``
-  * ``plantuml``
-  * ``textlive-full``
-
-`More details <https://github.com/rtfd/readthedocs-docker-images/blob/releases/3.x/Dockerfile>`__
+More details on software installed in images could be found in `rtfd/readthedocs-docker-images <https://github.com/rtfd/readthedocs-docker-images>`__ repository by browsing specific tag, marked in README as ``stable``.
```

Comment:
We didn't find a way to automate this, but is also a hard to read the docker file to see all the libs that are installed, maybe we should update to match the libs from latest (4.0) and stable (3.0)

Reply:
It's harder to maintain a list by copy-pasting it on every release. Dockerfile is pretty self-explanatory and easy to read: you even have one lib per line. I would leave this reference to the repository.